### ISSUE TITLE
Docker build fails at `COPY local` step 

### ISSUE THREAD

**Issue by derekcheong** [grafana/grafana#42953]
**What happened**:
Docker build failed with:
```
Step 20/49 : COPY ./local/* ./local/
COPY failed: no source files were specified
```

**What you expected to happen**:
Docker build should finish successfully.

**How to reproduce it (as minimally and precisely as possible)**:
Run `docker build --no-cache .`

**Anything else we need to know?**:
Looks like a PR request was raised covering this: https://github.com/grafana/grafana/pull/41315 but that has been closed without being merged.

**Environment**:
- Grafana version: `v8.3.0` and `v8.3.1`
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others: `Docker version 20.10.11, build dea9396`

**Comment by bolshoytoster** [comment]
This was fixed in #42390, if you clone the current state: `git clone --depth=1 https://github.com/grafana/grafana`, it should work.

In case it doesn't or you can't / don't want to you should just be able to add `/local` to `.dockerignore` file then remove or comment out `COPY ./local/* ./local/` from `Dockerfile` on line 30 as a temporary workaround.

(In the repo's root directory)
`echo /local >> .dockerignore`

`sed -i '/COPY .\/local\/\* .\/local\//d'`

**Comment by dsotirakis** [comment]
@derekcheong, closing the issue as it should have been fixed in #42390.
Thanks for raising it!
